scipy/fftpack/realtransforms.py

```diff
@@ -349,6 +359,22 @@ def dct(x, type=2, n=None, axis=-1, norm=None, overwrite_x=False):
     to a factor `2N`. The orthonormalized DCT-III is exactly the inverse of
     the orthonormalized DCT-II.
 
+    **Type IV**
+
+    There are several definitions of the DCT-IV; we use the following
+    (for ``norm=None``)::
+
+
+                N-1
+      y[k] = 2* sum x[n]*cos(pi*(2k+1)*(2n+1)/(4*N)), 0 <= k < N.
+                n=0
+
+    If ``norm='ortho'``, ``y[k]`` is multiplied by a scaling factor `f`::
+
+      f = 0.5*sqrt(2/N)
+
+      .. versionadded:: 1.2.0
```

Comment:
This should not be indented, it will not render properly:

https://7334-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.fftpack.dct.html#scipy.fftpack.dct